server/models/Update.js

```diff
@@ -121,6 +121,16 @@ export default function(Sequelize, DataTypes) {
 
       image: DataTypes.STRING,
 
+      isPrivate: {
+        type: DataTypes.BOOLEAN,
+        defaultValue: false,
+      },
+
+      contentStripped: {
+        type: new DataTypes.VIRTUAL(DataTypes.BOOLEAN),
```

Comment:
What is `DataTypes.VIRTUAL`? I can't find documentation on it.

Reply:
http://docs.sequelizejs.com/class/lib/data-types.js~VIRTUAL.html

It's used when the property is not backed by a database column.

Reply:
What's the benefit of exposing that in the model? (compared to having it only on GraphQL side).

Reply:
No benefit, I added it when I was manipulating with `stripContent`, so it can be removed now.

Reply:
I've removed it.